tendermint/tests/integration.rs

```diff
@@ -143,4 +144,30 @@ mod rpc {
         // For lack of better things to test
         assert_eq!(status.validator_info.voting_power.value(), 10);
     }
+
+    #[tokio::test]
+    #[ignore]
+    async fn event_subscription() {
+        let mut client = tendermint::rpc::event_listener::EventListener::connect(
+            "tcp://127.0.0.1:26657".parse().unwrap(),
+        )
+        .await
+        .unwrap();
+        let _ = client.subscribe(&"tm.event='Tx'".to_owned()).await.unwrap();
+        // let _ = client.subscribe(&"tm.event='NewBlock'".to_owned()).await.unwrap();
+
+        // Collect and throw away the response to subscribe
+        let _ = client.get_event().await.unwrap();
+
+        // Loop here is helpful when debuging parsing of JSON events
+        // loop{
+        let resp = client.get_event().await.unwrap();
+        dbg!(&resp);
+        // }
+        match resp {
+            Event::GenericStringEvent { data: _ } => assert!(false),
+            Event::GenericJSONEvent { data: _ } => assert!(true),
+            Event::JsonRPCTransactionResult { data: _ } => assert!(true),
```

Comment:
So this panics when we get a `Event::GenericStringEvent` but passes in all other cases?